playkit/src/main/java/com/kaltura/playkit/player/ExoPlayerWrapper.java

```diff
@@ -399,7 +409,15 @@ private MediaSource buildInternalExoMediaSource(PKMediaSourceConfig sourceConfig
                 mediaSource = new ProgressiveMediaSource.Factory(dataSourceFactory)
                         .createMediaSource(uri);
                 break;
-
+            case udp:
+                DataSource.Factory udpDatasourceFactory = () -> new UdpDataSource(3000, 100000);
```

Comment:
are these values for `maxPacketSize` and `socketTimeoutMillis` the preferred values?
Maybe we should expose as settings.

Reply:
will check this

Reply:
@OrenMe  done

Reply:
@noamtm  please review